<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DLG_HueSaturation</class>
 <widget class="QDialog" name="DLG_HueSaturation">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hue and Saturation</string>
  </property>
  <widget class="QSlider" name="slider_hue">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-180</number>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_hue">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>50</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-180</number>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_saturation">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>50</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-255</number>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_hue">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Hue</string>
   </property>
  </widget>
  <widget class="QSlider" name="slider_saturation">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-255</number>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_saturation">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Saturation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Okay</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
